core/src/test/java/org/hibernate/ogm/backendtck/queries/SimpleQueriesTest.java

```diff
@@ -553,6 +554,7 @@ public void testAutoFlushIsAppliedDuringQueryExecution() throws Exception {
 
 	@Test
 	@TestForIssue(jiraKey = "OGM-424")
+	@SkipByGridDialect(INFINISPAN_REMOTE)
```

Comment:
How come these tests are failing and the other aren't?

Reply:
Because the others verify results on a different transaction.